test/torchaudio_unittest/backend/sox_io/load_test.py

```diff
@@ -522,3 +522,14 @@ def test_frame(self, frame_offset, num_frames):
 
         assert sr == sample_rate
         self.assertEqual(expected, found)
+
+
+@skipIfNoSox
+class TestLoadNoSuchFile(PytorchTestCase):
+    def test_load_fail(self):
+        """
+        Must fail to load a non-existing file with a very specific message
+        """
+        path = "__No_SuCh_FiLe__"
```

Comment:
This looks very strange. Can we use an insignificant name, so that future maintainers focus naturally goes to test logic, instead of the file name.

```suggestion
        path = "non_existing_audio.wav"
```